### ISSUE TITLE
Flickering scrollbars on hackerrank site 

### ISSUE THREAD

**Issue by mariuskimmina** [brave/brave-browser#30475]
## Description 
I am experiencing flickering scrollbars on hackerrank.com challenges, see the video provided in *Actual result* for details.
When visiting the same site with firefox or chromium the scrollbars behave fine and there is no flickering.
When visitng the the same page in a brave private window without any extensions the issue remains and the scrollbars are still flickering.

## Steps to Reproduce
* start any challenge on hackerrank.com 
The one I used in the example: 
https://www.hackerrank.com/challenges/one-month-preparation-kit-mini-max-sum/problem?isFullScreen=true&h_l=interview&playlist_slugs%5B%5D=preparation-kits&playlist_slugs%5B%5D=one-month-preparation-kit&playlist_slugs%5B%5D=one-month-week-one

## Actual result:
https://github.com/brave/brave-browser/assets/38843153/cc396c3d-5a24-4042-85f5-abd763640cd0

## Expected result:
The scrollbars should not be flickering

## Reproduces how often: 
Everytime. It only happens in full screen mode.

## Brave version (brave://version info)
1.37.116 Chromium: 100.0.4896.127 (Official Build) (64-bit) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
Yes
- Can you reproduce this issue with the beta channel? 
Yes
- Can you reproduce this issue with the nightly channel? 
Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
No
- Does the issue resolve itself when disabling Brave Rewards? 
No
- Is the issue reproducible on the latest version of Chrome? 
Chromium does not reproduce this issue, I did not test it with Google chrome

**Comment by ryanbr** [comment]
Does it work work in private window mode @mariuskimmina

**Comment by mariuskimmina** [comment]
Using a private window does not change anything, it is still flickering. 
I noticed today tho that changing the "Gaps" setting on my deskop environment (PopOS) would make the flickering stop. 

![image](https://github.com/brave/brave-browser/assets/38843153/8aae4974-6ac7-45d0-9840-c597a72eb331)

And with changing this setting I could also reproduce this in chromium. So it does not seem to be an issue with brave after all.

**Comment by ryanbr** [comment]
I just noticed this on an older Brave build, can you update to the latest?

**Comment by mariuskimmina** [comment]
No it still happnes on: 1.51.118 Chromium: 113.0.5672.126 (Official Build) (64-bit) 

Here is a video showing that it is somehow related to the setting I mentioned in my last comment. This way it also happens when using chromium instead of brave - so I am not sure if this issue should remain open


https://github.com/brave/brave-browser/assets/38843153/1b369b84-7085-4e84-b124-1ccbbb3b0b2e

**Comment by ryanbr** [comment]
Probably open a ticket on https://bugs.chromium.org/ if that is the case @mariuskimmina   Maybe create a minimised test case?

**Comment by venugopal2003** [comment]
I had faced this issue of scrollbar flickering in Hackerrank. You need to set your browser screen to 100% resolution. It worked out for me.

**Comment by rishiraj88** [comment]
By 100%, you mean zoom, @venugopal2003 ?

**Comment by venugopal2003** [comment]
@rishiraj88 

Yeah! Zoom by 100 % then Hackerrank scrollbar issue gets resolved.
